[USING] rework Send (#733)
zkSNACKs/WasabiDoc: docs/using-wasabi/Send.md

1. Select the coins you want to spend.
2. Specify a destination address.
3. Label the observers of this transaction.

> 4. Specify the amount to send to the destination address, press the `Max` button to send a whole coin without change.

Reviewer: In step 1 we say `Select the coins`.
```suggestion
4. Specify the amount to send to the destination address, or press the `Max` button to send the whole coins.
```

Author: Yes?
but it doesn't matter which coins you have selected, pressing `Max` will send "a whole coin" [which might be confusing] without a change coin [to be more clear]

Reviewer: I think we should be talking about the selected coins not the output coin here.
Imagine if you were sending to many that sentence wouldn't be correct.

Author: Even in P2M, when the Max button is pressed, there is no change back to the user wallet, but the reminder goes to one of the destination outputs.

Reviewer: How about:
```suggestion
4. Specify the amount to send to the destination address, or press the `Max` button to send the whole selected amount.
```

## Coins

A coin is an unspent transaction output (UTXO), a chunk of bitcoin which can be sent in a future transaction.

> In the Wasabi wallet `Send` tab, you see a list of all the coins you can spend because.

Reviewer: ```suggestion
In the Wasabi wallet `Send` tab, you see a list of all the coins you can spend.
```

## Anonymity Set

> When you receive a bitcoin in a regular transaction with one input, and two outputs, then your coin can be linked to this one output.

Reviewer: You mean when you send?

Author: Well both...

But you are right, the example is more precise when sending / receiving a change.

```suggestion
When you send a bitcoin in a regular transaction with one input, and two outputs, then your change coin can be linked to this one input.
```

When sending bitcoin, you need to know the destination address of the receiver.
This commits to the spending condition that the receiver agrees to have for this coin.

> The address can be a public key hash [starting with `1...`], a script hash [starting with `3...`], or a native segwit bech32 public key hash [starting with `bc1q...`].

Reviewer: ```suggestion
The address can be a public key hash [starting with `1`], a script hash [starting with `3`], or a native segwit bech32 public key hash [starting with `bc1q`].
```

You can send a whole coin by selecting the `Max` button, which will build a transaction with only one output, the receiving address, and no change.
You can also see the current US Dollar value of the sending amount.

:::tip Avoid sending rounded values

> If you specify a rounded amount, like `0.0100 0000 bitcoin`, then the change output will not be rounded, like `0.0896 8413 bitcoin`.

Reviewer 2: ```suggestion
If you specify a rounded amount, like `0.01000000 bitcoin`, then the change output will not be rounded, like `0.0896 8413 bitcoin`.
```

Author: I think this is better readable... [it should be [in the wallet too](https://github.com/zkSNACKs/WalletWasabi/issues/2588), but that's OT]

This makes it easy for an observer to conclude that the spending amount was the `0.01 bitcoin`, and that the other output is the change back to the sender.

> So in order to increase your privacy, you can set a non-rounded amount, like `0.0101 6843`.

Reviewer 2: ```suggestion
So in order to increase your privacy, you can set a non-rounded amount, like `0.01016843`.
```

Once the transaction is signed, Wasabi will connect to a random Bitcoin P2P node over Tor and provide this transaction, then it will immediately disconnect.
This first node will gossip the transaction throughout the network, then miners can include it in a block.

If for some reason the first broadcast fails, then if you have Wasabi connected to [your own Bitcoin full node](/using-wasabi/BitcoinFullNode.md) this node will broadcast the transaction to the network.

> If this also fails, or you don't have a full node, then the transaction is send to the backend coordinator with a new Tor identity, who then gossips the transaction to the network.

Reviewer: ```suggestion
If this also fails, then the transaction is sent to the backend coordinator with a new Tor identity, who then broadcasts the transaction to the network.
```
